common/predictive-text/worker/model-compositor.ts

```diff
@@ -397,9 +397,7 @@ class ModelCompositor {
       context = models.applyTransform(postTransform, context);
     }
 
-    let wordbreaker = this.lexicalModel.wordbreaker || wordBreakers.default;
-    let postContextTokens = models.tokenize(wordbreaker, context); //.wordbreak(postContext);
-    let revertedPrefix = postContextTokens.left[postContextTokens.left.length - 1];
```

Comment:
I see that this has changed quite a bit in this final commit -- how did this just fall out with the refactor?

Reply:
`models.wordbreak` does exactly this.  Compare against the following code (from within the `models` namespace):  

https://github.com/keymanapp/keyman/blob/d2b08d6e7daf5c10685767f49cca678ed1d38550/common/models/templates/src/tokenization.ts#L76-L90

`tokenization.left.pop()` returns the same token.